sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormTrainingClient/FormTrainingClientTests.cs

```diff
@@ -61,6 +61,7 @@ public void GetCustomModelArgumentValidation()
 
             Assert.ThrowsAsync<ArgumentNullException>(() => client.GetCustomModelAsync(null));
             Assert.ThrowsAsync<ArgumentException>(() => client.GetCustomModelAsync(string.Empty));
+            Assert.ThrowsAsync<ArgumentException>(() => client.GetCustomModelAsync("1975-04-04"));
```

Comment:
What's special about 1975-04-04?

Reply:
[It's an easter egg!](https://en.wikipedia.org/wiki/Microsoft#1972%E2%80%931985:_The_founding_of_Microsoft)